examples/gallery/lines/linestrings.py

```diff
@@ -0,0 +1,50 @@
+# ruff: noqa: RUF003
+"""
+Plotting lines with LineString/MultiLineString geometry
+=====
```

Comment:
```suggestion
Plotting lines with LineString/MultiLineString geometry
=======================================================
```

Reply:
Same as above, but for commit https://github.com/GenericMappingTools/pygmt/pull/3711/commits/d222759b4041a982e730e88ba2da2ec1b057916f.